exercises/word-search/word-search.js

```diff
@@ -0,0 +1,52 @@
+//
+// This is only a SKELETON file for the 'Word Search' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+//
+
+function searchHorizontally() {
```

Comment:
All these top-level functions should _not_ be stubbed, as an implementation for this exercises does not necessarily need them. The `class` and `find` are correct.

Reply:
You can check corresponding [spec file](https://github.com/exercism/javascript/blob/4a0f58d67f27fda8d433f934aaeef386f3f1033e/exercises/word-search/word-search.spec.js) for the exercise and stub **only** the functions/classes used there.

Reply:
Thanks for your replies. Sorry for those mistakes, I didn't realize it at first.

I've had made those requested changes.